apps/api/src/main.ts

```diff
@@ -19,7 +34,7 @@ app.use(session({
   secret: 'top secret',
   cookie: {
     http: false,
-    sameSite: 'strict'
+    sameSite: 'lax'
```

Comment:
did it work before with strict? if so, how? 🤔

Reply:
@edunham OIDC workshop does not work with samesite strict. I used samesite strict for the base app because it could get away with it - there was no need to pass cookies around so I locked it down tight, however, for redirect auth, we do need cookie context, so it's changed to lax, which is the default samesite setting. it's still secure (as opposed to none).